src/components/PlayerView/index.tsx

```diff
@@ -125,7 +125,7 @@ export function PlayerView({
 
   useEffect(() => {
     const node = findNodeHandle(nativeView.current);
-    if (node) {
+    if (node && isFullscreenRequested) {
       dispatch('setFullscreen', node, isFullscreenRequested);
```

Comment:
I'm not sure about this change.
This means `setFullscreen` is not dispatched when `isFullscreenRequested` is set to `false`. This means you can not leave fullscreen once entered.

Maybe checking with `!== undefined` would solve this.

Reply:
Agree, `!== undefined` is probably what we want here. Please also make sure to do the same for `scalingMode`.

Reply:
Done, I didn't realize that typescript overloaded if with both null and false. It's logic in retrospect.
Done: d071e780aa5520610de20d9d53613109dd503bb8